### ISSUE TITLE
What is the minimal OS to run the official drmingw release?

### ISSUE THREAD

**Issue by asmwarrior** [jrfonseca/drmingw#81]
Hi, we have a discussion here:

[Re: error alert message when using ThreadSearch Plugin](https://forums.codeblocks.org/index.php/topic,25054.msg171178.html#msg171178)

in our Code::Blocks forum, one developer using Win7 64 bit can't not run the drmingw 0.9.7. Because some DLL files are missing.
I don't such issue in my Win7, because I maybe have some KBxxxxx installed which contains the necessary dlls.

Any ideas?

Thanks.

**Comment by jrfonseca** [comment]
It sounds the developer might be using a Windows 7 installation that lacks all updates from Microsoft.

I'm afraid I can't afford chasing problems with out of date Windows 7 installations.  I can't be sure that alone fixes the problem, but dealing with out of date Windows 7 installations presents too many unknowns, and there's no good reason to support it.

I know updating Windows 7 can be tricky, as I recently experienced, because a Windows 7 freshly installed from an ISO is not able to automatically download updates from Microsoft, and one needs to follow the steps described [here](https://superuser.com/a/996072) to bootstrap.

But the above difficulties are not an excuse not to do it.  It's imperative folks using Windows 7 do so, not only for better security, but also so we have all comparable systems.

In short, to answer your question, the minimal OS to run drmingw releases is _Windows 7 SP1, plus Platform Update, and all other updates from Microsoft Update._  Anything older is unsupported.

If the issue still reproduces after installing all Windows 7 updates, then let me know, and I'll look into it.

**Comment by asmwarrior** [comment]
I understand your option, and thanks for the explanation.
